src/http.ts

```diff
@@ -42,46 +42,52 @@ export type DevicesResponse = { devices: DevicesByName } | ErrorResponse;
 export type QMIResponse = { qmi: QMI } | ErrorResponse;
 export type QMIsResponse = { qmis: QMI[] } | ErrorResponse;
 
-export async function _request(path: string, data: any = {}, method: string) {
+type PromiseRequestCallback = (ok: (value?: any) => void, err: (value?: any) => void) => request.RequestCallback;
```

Comment:
I know we don't specify this in our tslint but I've been following a max-line-length of 100. 120 is also fine if you prefer it, in which case this line is good.